web-client/iron-remote-gui/src/services/wasm-bridge.service.ts

```diff
@@ -57,6 +57,11 @@ export class WasmBridgeService {
     sessionObserver: Observable<SessionEvent> = this.sessionEvent.asObservable();
     scaleObserver: Observable<ScreenScale> = this.scale.asObservable();
 
+    dynamicResize = new Subject<{
+        width: number;
+        height: number;
+    }>();
```

Comment:
suggestion: Any reason for using a `Subject` instead of an `Observable` here?

Reply:
Observable constructor require a callback, where with subject I could simply do dynamicResize.next.

Reply:
I see!! Can you add that as an in-code comment, please?